net/udpbroadcastrelay/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		os-udpbroadcastrelay
-PLUGIN_VERSION=		0.6
+PLUGIN_VERSION=		0.7
 PLUGIN_DEVEL=         YES
```

Comment:
You need 
`PLUGIN_DEPENDS= udpbroadcastrelay`

Otherwise it will not be installed when installing the plugin

Reply:
Indeed, my commit to add that went out seconds before your comment :)

Squashed into one commit now.